packages/core/src/codewhispererChat/editor/context/file/importReader.ts

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { Java, Python, TypeScript, Tsx } from '@aws/fully-qualified-names'
+import { Java, Python, TypeScript } from '@aws/fully-qualified-names'
```

Comment:
```suggestion
import { Java, Python } from '@aws/fully-qualified-names'
```

    npm ERR! src/codewhispererChat/editor/context/file/importReader.ts(6,24): error TS6133: 'TypeScript' is declared but its value is never read.